### PR TITLE
[fix] Fix the bug that "hideTitleBar" doesn't work

### PR DESCRIPTION
**Pull request by MrPeak**
"panel.setTitlebarAppearsTransparent(true)" only make titlebar transparent, but won't hide it.

### CODE REVIEW

**Comment by mathieudutour** [comment]
Nice one! Can you add `NSWindowTitleHidden` to the list of globals at the top of the file so that eslint is happy?

**Comment by MrPeak** [comment]
Already done 🤓

**Comment by MrPeak** [comment]
BTW

The existing configurations is still not meet my needs, I will make more pull requests...

For example:

- [ ] Hide whole titlebar（include closeButton.） 
- [ ] Load remote page
- [ ] ...

File: lib/index.js [hunk 1]
```diff
@@ -33,6 +33,10 @@ function WebUI (context, htmlName, options) {
   panel.setTitle(options.title || context.plugin.name())
   if (options.hideTitleBar) {
     panel.setTitlebarAppearsTransparent(true)
+    panel.setTitleVisibility(
+    
+    
+    )
```
Comment (mathieudutour) [inline lib/index.js]: there is something weird here

**Pull request merged**